README.md

```diff
@@ -5,9 +5,9 @@
 
 ## Description
 
-Red Datasets provides classes that provide common datasets such as iris dataset.
+Red Datasets provides an easy way to fetch standard datasets commonly used in the data science community.
 
-You can use datasets easily because you can access each dataset with multiple ways such as `#each` and Apache Arrow Record Batch.
+You can access these datasets in several formats, including Arrays, Numo::NArray, and ApacheArrow RecordBatch.
```

Comment:
Why did you remove `#each`?
This is the important interface of this rather than `Array`, Numo::NArray, Apache Arrow Record Batch and so on.

Reply:
Please don't change "Apache Arrow Record Batch" notation.
"ApacheArrow" (no space) is invalid.

Reply:
I got it.

Reply:
I watched the video. https://www.youtube.com/watch?v=PI2M6qeHi_g
And I found that you think "each" is important.

Reply:
But I do not fully agree with that opinion. Yes, I admit that working with natural integration with the Ruby syntax is great. But what people really want to do is get a dataset with minimal effort. But, I do not have enough knowledge and time to investigate and tackle this difficult issue seriously, so I will withdraw...

Reply:
Reverted
The text I changed has a different meaning than before. This is not good.